packages/database/src/migrations/20210623220403-LESMISGrossIntakeRate-modifies-data.js

```diff
@@ -74,9 +74,9 @@ const generateProvinceReportConfig = ({ educationLevel }) => {
       'keyValueByDataElementName',
       {
         transform: 'select',
-        "'Male'": `$row.gir_district_${educationLevel}_m`,
-        "'Female'": `$row.gir_district_${educationLevel}_f`,
-        "'Total'": `$row.gir_district_${educationLevel}_t`,
+        "'Male'": `exists($row.gir_district_${educationLevel}_m) ? $row.gir_district_${educationLevel}_m/100 : undefined`,
+        "'Female'": `exists($row.gir_district_${educationLevel}_f) ? $row.gir_district_${educationLevel}_f/100 : undefined`,
+        "'Total'": `exists($row.gir_district_${educationLevel}_t) ? $row.gir_district_${educationLevel}_t/100 : undefined`,
```

Comment:
@rohan-bes we should probably turn this into an internal `divide` function that gets attached to mathjs right?

Reply:
Such a function does actually already exist in this PR: https://github.com/beyondessential/tupaia/pull/2624
I've been horrible at getting that PR merged as it failed testing a little while back and I haven't got around to investigating the failure as yet, I'll prioritise it ASAP

Reply:
Alternatively, if @EMcQ-BES wants to cherry pick 4e2b0f46c733c8c47a53b20302d10fa8ff578d03 and f1a807eb4d5d909467bc546f6c646f8674f1433c then we can work out a merging strategy based on whoever merges first?

Reply:
@EMcQ-BES merge this now, and then on Wednesday when we return to it we can see if @rohan-bes's work has made it in to dev and either use that or cherry pick it at that point